src/shared/mount-setup.c

```diff
@@ -66,7 +66,7 @@ static const MountPoint mount_table[] = {
           NULL,          MNT_FATAL|MNT_IN_CONTAINER|MNT_FOLLOW_SYMLINK },
         { "sysfs",       "/sys",                      "sysfs",      NULL,                                      MS_NOSUID|MS_NOEXEC|MS_NODEV,
           NULL,          MNT_FATAL|MNT_IN_CONTAINER },
-        { "devtmpfs",    "/dev",                      "devtmpfs",   "mode=755" TMPFS_LIMITS_DEV,               MS_NOSUID|MS_STRICTATIME,
+        { "devtmpfs",    "/dev",                      "devtmpfs",   "mode=755",                                MS_NOSUID|MS_STRICTATIME,
```

Comment:
This would also remove the size restriction. Devices and symlinks don't take any space, so there shouldn't be any need to not limit the size. Regular files shouldn't exist there either.

Reply:
I agree. I don't think there's a reason to drop the size limit.